web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -162,11 +254,35 @@ const FosterQuestionnaire = ({
           </Stack>
           <ModalHeader>Foster Questionnaire</ModalHeader>
           <Flex paddingX={6} flexDir="column" gap={4} overflowY="scroll">
-            {data[fosterType].map((question) => {
+            {data[fosterType].map(({ key, title }) => {
               return (
-                <Flex key={question.key} flexDir="column" gap={2}>
-                  <Text>{question.title}</Text>
-                  <Input />
+                <Flex key={key} flexDir="column" gap={2}>
```

Comment:
Verify that the comments above are consistent for the mobile view as well - maybe we should revisit refactoring this code so that we don't have repeated parts like this? You can make it a TODO comment since it's outside the scope of the ticket.